test/util.ts

```diff
@@ -20,6 +21,14 @@ export function mocked<T extends object>(module: T): jest.Mocked<T> {
   return jest.mocked(module);
 }
 
+/**
+ * Simple wrapper for getting mocked version of a module
+ * @param module module which is mocked by `jest-mock-extended.mockDeep`
+ */
+export function mockedDeep<T extends object>(module: T): DeepMockProxy<T> {
```

Comment:
Use different name to not clash naming with `mockDeep` from `jest-mock-extended`

Reply:
How about `mockedExtended`, alluding to the fact that this function uses the `mockDeep` function from `jest-mock-**extended**`? I also realized that `mocked()` always uses a deep mock, so there's no need to add `deep` to the name here either. See 34fae8521fae897d27118cce12f90289fb435c02.